arangod/RocksDBEngine/RocksDBHotBackup.cpp

```diff
@@ -181,6 +227,8 @@ void RocksDBHotBackup::statId(std::string const& id, VPackBuilder& result, bool
         VPackArrayBuilder a(&result);
         result.add(VPackValue(id));
```

Comment:
is this id field now redundant to meta information's id field?

Reply:
Most likely yes, however there is not way to reliably check whether one found all places that call this api. Therefore leave it as it is.

This shows that we need a more grpc like approach for api's. We need a clear typed interface for those rpc and let the compiler figure out that something is wrong.